tests/frontend/tests/test_workloads.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from random import randint
+from tests.frontend.pages.workloads.workloads import workloads
+from tests.frontend.tests.base_tests import BaseTest
+
+
+@pytest.mark.integration
+class WorkloadsTests(BaseTest):
+    def setUp(self):
+        super().setUp()
+        self.workloads = workloads(self.driver)
+        self.workloads.load()
+
+    def test01_delete_selected_workloads(self):
+        """
```

Comment:
please add test description like:
Test case for creating a workload and deleting it.